setup.py

```diff
@@ -1,21 +1,21 @@
 #!/usr/bin/env python
 
 import os
+import re
 from setuptools import setup
-from pip.req import parse_requirements
 
 
 def read(file_name):
     return open(os.path.join(os.path.dirname(__file__), file_name)).read()
```

Comment:
Not sure if this will close the file. If it does, great! if not, please use something like:
```python
with open(os.path.join(os.path.dirname(__file__), file_name), 'r') as f:
  return f.read()
```